src/app.ts

```diff
@@ -65,6 +67,16 @@ const defaultModule = {
   startIdpMetadataUpdater
 };
 
+const cacheDuration = `${container.resolve(CACHE_MAX_AGE_SECONDS)} seconds`;
+
+const cachingMiddleware = apicache.options({
+  debug: false,
```

Comment:
```suggestion
  debug: process.env.NODE_ENV === "development" || process.env.APICACHE_DEBUG === "true",
```

also add `APICACHE_DEBUG` to the README, thanks

Reply:
fixed in 4394920